api/enterprise-api-ref/personal-groups.markdown

```diff
@@ -51,16 +46,37 @@ The personal groups API enables creating host groups based on host filters (the
         "operator":"value2"
       }
     },
-    "hostsFilter": {
-      "includes": {
-        "type": "hostkey",
-        "data": ["SHA=first1"]
+    "hostFilter": {
```

Comment:
Is this fairly large example repeated a few times? Maybe we could just do it once up-front and then refer to it later.

Also, showing some sample results that take advantage of each item in the example request would be nice.